### DISCUSSION TITLE
Welcome to Data Loader Discussions!

### DISCUSSION THREAD

**Discussion by ashitsalesforce** [forcedotcom/dataloader#736] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by souvsark** [comment]
First of all i would like to be honest - i am not a software guy and i do not have technical background. I am a business analyst and working in automobile domain. Recently i was going through Salesforce admin course and there i came across a concept of dataloader. Now i have a linux OS in my personal laptop and i am struggling to install dataloader in it. I have watched videos but somewhere i am missing out something i have Linuxmint installed in the machine. can anybody please help ?

**Reply by ashitsalesforce** [reply]
Opening discussion [here](https://github.com/forcedotcom/dataloader/discussions/847) for this question.
